src/sentry/integrations/vsts/notify_action.py

```diff
@@ -1,42 +1,77 @@
 from __future__ import absolute_import
 
 import logging
+import six
 
 from django import forms
 
+from sentry.models.integration import Integration
 from sentry.rules.actions.base import TicketEventAction
+from sentry.shared_integrations.exceptions import IntegrationError
 from sentry.utils.http import absolute_uri
 from sentry.web.decorators import transaction_start
 
+from sentry.integrations.vsts.integration import VstsIntegration
+
 logger = logging.getLogger("sentry.rules")
 
 INTEGRATION_KEY = "integration"
 
 
 class AzureDevopsNotifyServiceForm(forms.Form):
```

Comment:
@ceorourke I realize we could probably make an abstract base class for `AzureDevopsNotifyServiceForm` that we use for Jira as well since I think the functions are pretty much the same, yea?